src/chrome/chromeDebugSession.ts

```diff
@@ -82,14 +97,28 @@ export class ChromeDebugSession extends LoggingDebugSession {
             return errMsg;
         };
 
+        const reportErrorTelemetry = (err, exceptionType: ExceptionType)  => {
+            let properties: ITelemetryPropertiesPayload = {};
+            properties.successful = "false";
+            properties.exceptionType = exceptionType;
+            properties.timeTakenInMilliseconds = "unknown";
```

Comment:
Should this just be empty instead of a string that can't be parsed as a number?

Reply:
It's changed to empty.